http/server.go

```diff
@@ -529,6 +530,25 @@ func (h *DrandHandler) Health(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(b)
 }
 
+func (h *DrandHandler) ChainHashes(w http.ResponseWriter, r *http.Request) {
+	chainHashes := make([]string, 0)
+	for chainHash := range h.beacons {
+		if chainHash != common.DefaultChainHash {
+			chainHashes = append(chainHashes, chainHash)
+		}
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Cache-Control", "no-cache")
+
+	resp := make(map[string][]string)
+	resp["chains"] = chainHashes
```

Comment:
is there a reason for the top level 'chains':[]string formatting? could we just return the array of chains directly?

Reply:
No, no specific reason. If you prefer that, I can change it.